desktop/core/src/desktop/middleware.py

```diff
@@ -633,7 +629,7 @@ def clean_username(self, username, request):
     return username
 
 
-class SpnegoMiddleware(object):
+class SpnegoMiddleware(MiddlewareMixin):
```

Comment:
Please skip this one as @wing2fly has a PR with some testing

Reply:
done